packages/nimble-components/src/card/template.ts

```diff
@@ -1,4 +1,9 @@
 import { html } from '@microsoft/fast-element';
 import type { Card } from '.';
 
-export const template = html<Card>`<slot></slot>`;
+export const template = html<Card>`
+    <section aria-labelledby="titleSlot">
+        <slot name="title" id="titleSlot"></slot>
+        <slot></slot>
+    </section>
```

Comment:
noticed in the [full page accessibility tree view](https://developer.chrome.com/blog/full-accessibility-tree/#full-accessibility-tree-in-devtools) that the increment and decrement buttons for the number field are showing up as focusable which they aren't (can't tab to them or have a focus state on them). It's separate from this PR but can you create an issue for those:
![image](https://github.com/ni/nimble/assets/1588923/42ae7458-e947-4ae3-b9da-ad8ac94aefd1)

Reply:
Created an issue here: #1644